### ISSUE TITLE
Intermittent failure of /_mozilla/webgl/conformance-1.0.3/conformance/more/conformance/quickCheckAPI-S_V.html

### ISSUE THREAD

**Issue by KiChjang** [servo/servo#10698]
```
  ▶ OK [expected CRASH] /webgl/conformance-1.0.3/conformance/more/conformance/quickCheckAPI-S_V.html

  ▶ Unexpected subtest result in /webgl/conformance-1.0.3/conformance/more/conformance/quickCheckAPI-S_V.html:
  │ FAIL [expected PASS] WebGL test #0: testValidArgs
  │   → assert_true: testValidArgs expected true got false
  │ 
  │ reportTestResultsToHarness/<@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/more/unit.js:900:7
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1381:20
  │ test@http://web-platform.test:8000/resources/testharness.js:495:9
  │ reportTestResultsToHarness@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/more/unit.js:899:1
  │ runTests@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/more/unit.js:188:5
  │ initTests@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/more/unit.js:922:5
  └ @http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/more/unit.js:947:7
```

**Comment by frewsxcv** [comment]
Duplicate of https://github.com/servo/servo/issues/10698

**Comment by KiChjang** [comment]
#10656 is `quickCheckAPI-G_I.html`; this is `quickCheckAPI-S_V.html`.

**Comment by frewsxcv** [comment]
Woops, sorry about that.

**Comment by nox** [comment]
Merged with ##10656.
